addons/sale_crm/models/crm_lead.py

```diff
@@ -87,6 +73,19 @@ def action_view_sale_order(self):
             action['res_id'] = orders.id
         return action
 
+    def _get_opportunity_quotation_context(self):
```

Comment:
Let's go with "_prepare_opportunity_quotation_context".

We usually use the "prepare" keyword for methods that prepare a dict of values.

Side note: Could add a bit of docstring to help.
Something like:

""" Prepares the context for a new quotation (sale.order) by sharing the values of common fields. """"